feat(changelog): functions-added-serverless-network-metrics 2024-11-05 (#3944)
scaleway/docs-content: changelog/november2024/2024-11-05-functions-added-serverless-network-metrics.mdx

---

> title: Serverless network metrics

Reviewer: ```suggestion
title: Serverless network metrics are evolving
```

status: added
author:
    fullname: 'Join the #serverless-functions channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-11-05
category: serverless
product: functions
---

> In the metrics panel for Serverless Functions, we have added access to network information on data received and transmitted.

Reviewer 2: ```suggestion
In the metrics panel for Serverless Functions, we added access to network information on data received and transmitted.  Refer to the [dedicated documentation](/serverless/functions/how-to/monitor-function/) for more information on how to monitor a function.
```

> Metrics are evolving, in Serverless Functions metrics panel we added access to network informations on received data and transmitted data.

Reviewer: ```suggestion
In the Serverless Functions metrics panel we have added access to network information on received and transmitted data.
```
